src/Directory/Biobanks.Web/Controllers/SuperUserController.cs

```
@@ -43,18 +44,24 @@ public async Task<ViewResult> SearchIndex()
             // Cluster Health Status
             ViewBag.Status = await _indexService.GetClusterHealth();
 
-            // Index Sample Counts
-            var model = new SearchIndexModel
+            // View Model (Default View Shows No Data)
+            var model = new SearchIndexModel();
+
+            // Prevent Error Propagation When Search Index Is Down
```

I don't like resolving this by hiding it (the empty `catch`).

If search is down or uncontactable, we want it to log, and errors to show up in app insights etc...

not sure exactly what I want to see though.

Users probably should get a friendly message that the operation failed (rather than the stats presumably all returning 0?), but not like the current 500 error page...

---

maybe for the sake of the PR we leave this, but be sure to create a PBI to address it

---

PBI 30764